### DISCUSSION TITLE
in menu component escape not work correctly

### DISCUSSION THREAD

**Discussion by jsb1992s** [primefaces/community#1555] [PrimeNG]
Custom Content in menu component not show html correctly :
for example, there is this problem in the demo code (https://stackblitz.com/run?file=src%2Fapp%2Fdemo%2Fmenu-custom-content-demo.ts) it does not parse html code of label

**Comment by cetincakiroglu** [comment]
Hi,

Example link is broken it redirects to the stackblitz.com

Could you please update the link so we can investigate the issue?
